test/e2e/node/kubelet.go

```diff
@@ -271,7 +271,7 @@ var _ = SIGDescribe("kubelet", func() {
 		ns string
 	)
 	f := framework.NewDefaultFramework("kubelet")
-	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelBaseline
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
```

Comment:
this is fine if needed for 1.24, but do all the tests within this require privileged, or only the nfs ones?

/lgtm
/approve

Reply:
Like below: 
![image](https://user-images.githubusercontent.com/2010320/161281348-162d2d02-641d-4ccf-9368-f4eb2c406ec9.png)